### PR TITLE
add elk project

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

**Comment by dymurray** [comment]
ACK

File: elk-ansibleapp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM fusordevel/ansibleapp-base
```
Comment (dymurray) [inline elk-ansibleapp/Dockerfile]: Pull from ansibleapp/ansibleapp-base now that we have autobuilding on this repo.

File: elk-ansibleapp/ansible/container.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - 5601:5601
+    expose:
+      - 5601
+    user: kibana
+    links:
+    - elasticsearch
+    working_dir: /usr/share/kibana/bin
+    command: [./kibana]
+
+registries:
```
Comment (dymurray) [inline elk-ansibleapp/ansible/container.yml:54]: Make this point to ansibleapp perhaps?

File: elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+- oso_deployment:
+    project_name: elk-ansibleapp
+    labels:
+      app: elk-ansibleapp
+      service: elasticsearch
+    deployment_name: elasticsearch
+    containers:
+    - securityContext: {}
+      name: elasticsearch
+      image: docker.io/fabianvf/elk-ansibleapp-elasticsearch:20170131194913
```
Comment (dymurray) [inline elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml]: This needs to point to ansibleapp repo and these containers need to be pushed up with latest tag

File: elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+      app: elk-ansibleapp
+      service: kibana
+    deployment_name: kibana
+    containers:
+    - securityContext: {}
+      name: kibana
+      workingDir: /usr/share/kibana/bin
+      args:
+      - ./kibana
+      image: docker.io/fabianvf/elk-ansibleapp-kibana:20170131194913
```
Comment (dymurray) [inline elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml]: same as above

File: elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+- oso_deployment:
+    project_name: elk-ansibleapp
+    labels:
+      app: elk-ansibleapp
+      service: logstash
+    deployment_name: logstash
+    containers:
+    - securityContext: {}
+      name: logstash
+      image: docker.io/fabianvf/elk-ansibleapp-logstash:20170131194913
```
Comment (dymurray) [inline elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main.yml]: Same as above

File: elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main_170131141453.yml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+- oso_deployment:
+    project_name: elk-ansibleapp
+    labels:
+      app: elk-ansibleapp
+      service: elasticsearch
+    deployment_name: elasticsearch
+    containers:
+    - securityContext: {}
+      name: elasticsearch
+      image: docker.io/fabianvf/elk-ansibleapp-elasticsearch:None
```
Comment (dymurray) [inline elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main_170131141453.yml]: Right idea just point this to ansibleapp and push that image to ansibleapp

Comment (fabianvf) [reply elk-ansibleapp/ansible/roles/elk-ansibleapp-openshift/tasks/main_170131141453.yml]: Oh this file definitely shouldn't have been committed

**Comment by dymurray** [comment]
ACK

**Pull request merged**